src/Helper/Quote.php

```diff
@@ -240,14 +234,17 @@ public function getValidatedShippingMethods()
                 $this->log->debug('Shipping method error: ' . $shipMethod->getErrorMessage());
                 continue;
             }
+
+            $priceIncludesTaxes = $this->taxData->shippingPriceIncludesTax($this->quote->getStore());
```

Comment:
use different variable name for safety and readability (something to do with shipping?)